### ISSUE TITLE
Data range problem

### ISSUE THREAD

**Issue by lzmmq** [voletiv/mcvd-pytorch#32]

**Comment by lzmmq** [comment]
my concern is Without normalization → Training may be unstable due to large variations in scale, leading to vanishing or exploding gradients.
With normalization → It may alter the data distribution, potentially losing important physical meanings and relationships.
